locales/fr-FR.json

```diff
@@ -321,7 +321,7 @@
       "push_notifications": {
         "alerts": {
           "favourite": "Messages aimés",
-          "follow": "Nouveaux abonné·e·s",
+          "follow": "Nouveaux abonnés",
```

Comment:
This one isn't gender neutral

Reply:
I know using "account" instead of "user" isn't a 100% satisfying solution but French being what it is, it's the closest translation we can get without using the middle dot, which is not a 100% satisfying solution either for big words in UIs (in my opinion). As I said I'm using the official Québec recommendation which advise first to use words that have defined genders or not gendered but can describe the whole group of persons, and use the middle dot if it's not possible.
But it's also closer to the reality, since an account does not exactly equals user on social networks. A Fediverse account can have no user behind it (bot) or many users (company). So in the end it's not that annoying.

Reply:
Sounds like the best compromise anyways for inclusivity 👍